RELEASE.md

```diff
@@ -8,6 +8,18 @@
 
 In the below example, we will pretend that we're upgrading from `0.11.0` to `0.12.0`. Please use the CORRECT release numbers as these are just example numbers.
 
+## Release timeline
+
+Usually we start a release process on either Monday or Tuesday and finish by Friday.
+
+Here's our usual schedule:
+* **Monday (Notify):** 48-hour notification. Communicate to maintainers and public channels a release will be cut on Wednesday and to merge any pending PRs. Inform QE team. Start work on blog post as it is usually the longest part of the release process.
+* **Tuesday (Pre-release Testing & Blog):** Test the pre-release (master branch) build briefly. Get feedback from QE (if applicable). Push the blog post for review (as it usually takes a few back-and-forth reviews on documentation).
+* **Wednesday (Release):** Cut the new release using the below release process. Will take 24 hours for release to hit all release channels (Flatpak, Brew, Chocolatey, etc.).
+* **Thursday (Post-release Testing & Blog):** Test the post-release build briefly for any critical bugs. Confirm that new release has been pushed to all release channels. Push the blog post live.
+* **Friday (Communicate):** Friday is statistically the best day for new announcements. Post on internal channels. Post on reddit, hackernews, twitter, etc.
```

Comment:
Maybe also worth noting that we want to be done by Friday and not have any surprises leading into the weekend.